### PR TITLE
Simplify config and default to all grammars

### LINKED ISSUE #4: Sass?
Will there be Sass support? only scss works..

### PR DESCRIPTION
**Pull request by vinkla**
This pull request simplifies the configuration by defaulting to all available CSS grammars. This way newcomers/beginners don't have to add custom grammars. The package will "just works".

This closes #4

### CODE REVIEW

File: src/main.js [hunk 1]
```diff
@@ -1,20 +1,19 @@
...
   }
   getGrammarScopes () {
-    const { additionalGrammars, lessSupport, scssSupport } = atom.config.get('ide-css')
-    return ['source.css']
-      .concat(lessSupport ? 'source.css.less' : [])
-      .concat(scssSupport ? 'source.css.scss' : [])
-      .concat(additionalGrammars || [])
+    return [
+      'source.css',
+      'source.sass',
```
[the remark this answers is not in the document]
Comment (liuderchi) [reply src/main.js]: @vinkla thanks for your experiment

I'm wondering how does the outline view look like when opening a `foo.sass` file? 

in mine says _"No outline available"_

---

I used to check the vscode-css-languageserver-bin README but looking deeper then found it supports only `css`, `less`, `scss` grammars in its setting

https://github.com/vscode-langservers/vscode-css-languageserver/blob/a2e637feb1aba9ad228aebd7f59461df00d4ef38/src/cssServerMain.ts#L19-L21

Also, assuming vscode-css-languageserver-bin is used in vscode, I open `foo.sass` file in vscode but found no syntax-highlight/auto-compolent support

[the remark this answers is not in the document]
Comment (vinkla) [reply src/main.js]: Did some more testing and outline view doesn't seem to work for `.sass` files. Though, it doesn't seem to produce any errors.

File: package.json [hunk 2]
```diff
@@ -34,11 +34,6 @@
   "engines": {
     "atom": ">=1.21.0"
   },
-  "enhancedScopes": [
```
[the remark this answers is not in the document]
Comment (vinkla) [reply package.json]: I've pushed an update.

File: src/main.js [hunk 3]
```diff
@@ -1,20 +1,18 @@
...
   getGrammarScopes () {
-    const { additionalGrammars, lessSupport, scssSupport } = atom.config.get('ide-css')
-    return ['source.css']
-      .concat(lessSupport ? 'source.css.less' : [])
-      .concat(scssSupport ? 'source.css.scss' : [])
-      .concat(additionalGrammars || [])
+    return [
+      'source.css',
+      'source.css.less',
+      'source.css.scss'
```
Comment (liuderchi) [inline src/main.js]: Please help adding comma here to pass eslint check

Comment (vinkla) [reply src/main.js]: Fixed.

**Pull request merged**